src/TimeSteppers/TimeSteppers.jl

```diff
@@ -24,6 +27,23 @@ function update_state! end
 function compute_tendencies! end
 function compute_flux_bc_tendencies! end
 
+"""
+    $SIGNATURES
+
+Defines initialization routines for the first call to `update_state!` after initialization. Defaults to invoking `update_state!(model; kw...)`.
+"""
+function initialization_update_state!(model::AbstractModel; kw...) 
+    initialize!(model)
+    update_state!(model; kw...) # fallback
+end
```

Comment:
I think we will be able to get rid of `initialization_update_state!` soon FYI. It was introduced to support work with Reactant but I do not think it is needed anymore.